examples/complete/vpc.tf

```diff
@@ -28,7 +28,7 @@ module "vpc" {
   public_subnets_cidr_blocks         = var.public_subnets_cidr_blocks
   # Create subnets flag
   create_public_subnets         = true
-  enable_nat_gateway            = true
```

Comment:
If we left this as true, would we need the workaround to pull down the script with the AWS CLI?

Reply:
Not really, it will work with the s3 URL. But with AWS CLI, it will work on both cases.